<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Failed suite [Failed suite [Failed suite [Failed suite [Failed suite [all-tests-in-parallel]]]]]" duration-ms="104177" started-at="2014-08-07T04:58:32Z" finished-at="2014-08-07T05:00:17Z">
    <groups>
    </groups>
    <test name="parallel-ui-tests(failed)(failed)(failed)(failed)(failed)" duration-ms="104177" started-at="2014-08-07T04:58:32Z" finished-at="2014-08-07T05:00:17Z">
      <class name="teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest">
        <test-method status="PASS" signature="classSetup()[pri:0, instance:teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest@dc6233]" name="classSetup" is-config="true" duration-ms="9348" started-at="2014-08-07T04:58:33Z" finished-at="2014-08-07T04:58:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- classSetup -->
        <test-method status="FAIL" signature="allTests()[pri:0, instance:teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest@dc6233]" name="allTests" duration-ms="94049" started-at="2014-08-07T04:58:42Z" finished-at="2014-08-07T05:00:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<[The interval 1 - 5 is not divisible by the specified increment.]> but was:<[Please enter valid numbers for all the options.]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<[The interval 1 - 5 is not divisible by the specified increment.]> but was:<[Please enter valid numbers for all the options.]>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest.testInputValidationForNumScaleQuestion(InstructorFeedbackEditPageUiTest.java:627)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest.allTests(InstructorFeedbackEditPageUiTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- allTests -->
        <test-method status="PASS" signature="classTearDown()[pri:0, instance:teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest@dc6233]" name="classTearDown" is-config="true" duration-ms="0" started-at="2014-08-07T05:00:16Z" finished-at="2014-08-07T05:00:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- classTearDown -->
      </class> <!-- teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest -->
      <class name="teammates.test.cases.ui.browsertests.InstructorFeedbackSubmitPageUiTest">
        <test-method status="PASS" signature="classSetup()[pri:0, instance:teammates.test.cases.ui.browsertests.InstructorFeedbackSubmitPageUiTest@332aff]" name="classSetup" is-config="true" duration-ms="25343" started-at="2014-08-07T04:58:33Z" finished-at="2014-08-07T04:58:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- classSetup -->
        <test-method status="FAIL" signature="testAll()[pri:0, instance:teammates.test.cases.ui.browsertests.InstructorFeedbackSubmitPageUiTest@332aff]" name="testAll" duration-ms="28395" started-at="2014-08-07T04:58:58Z" finished-at="2014-08-07T04:59:27Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:267)
	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:259)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackSubmitPageUiTest.testSubmitAction(InstructorFeedbackSubmitPageUiTest.java:188)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackSubmitPageUiTest.testAll(InstructorFeedbackSubmitPageUiTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAll -->
        <test-method status="PASS" signature="classTearDown()[pri:0, instance:teammates.test.cases.ui.browsertests.InstructorFeedbackSubmitPageUiTest@332aff]" name="classTearDown" is-config="true" duration-ms="0" started-at="2014-08-07T04:59:27Z" finished-at="2014-08-07T04:59:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- classTearDown -->
      </class> <!-- teammates.test.cases.ui.browsertests.InstructorFeedbackSubmitPageUiTest -->
      <class name="teammates.test.cases.ui.browsertests.StudentFeedbackSubmitPageUiTest">
        <test-method status="PASS" signature="classSetup()[pri:0, instance:teammates.test.cases.ui.browsertests.StudentFeedbackSubmitPageUiTest@656185]" name="classSetup" is-config="true" duration-ms="16885" started-at="2014-08-07T04:58:33Z" finished-at="2014-08-07T04:58:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- classSetup -->
        <test-method status="FAIL" signature="testAll()[pri:0, instance:teammates.test.cases.ui.browsertests.StudentFeedbackSubmitPageUiTest@656185]" name="testAll" duration-ms="26978" started-at="2014-08-07T04:58:50Z" finished-at="2014-08-07T04:59:17Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:267)
	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:259)
	at teammates.test.cases.ui.browsertests.StudentFeedbackSubmitPageUiTest.testSubmitAction(StudentFeedbackSubmitPageUiTest.java:207)
	at teammates.test.cases.ui.browsertests.StudentFeedbackSubmitPageUiTest.testAll(StudentFeedbackSubmitPageUiTest.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAll -->
        <test-method status="PASS" signature="classTearDown()[pri:0, instance:teammates.test.cases.ui.browsertests.StudentFeedbackSubmitPageUiTest@656185]" name="classTearDown" is-config="true" duration-ms="0" started-at="2014-08-07T04:59:17Z" finished-at="2014-08-07T04:59:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- classTearDown -->
      </class> <!-- teammates.test.cases.ui.browsertests.StudentFeedbackSubmitPageUiTest -->
      <class name="teammates.test.cases.logic.EvaluationsLogicTest">
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:teammates.test.cases.logic.EvaluationsLogicTest@7cb926]" name="testSetUp" is-config="true" duration-ms="546" started-at="2014-08-07T04:58:32Z" finished-at="2014-08-07T04:58:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testSetup()[pri:0, instance:teammates.test.cases.ui.browsertests.InstructorCommentsPageUiTest@1187658]" name="testSetup" is-config="true" duration-ms="0" started-at="2014-08-07T04:58:33Z" finished-at="2014-08-07T04:58:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetup -->
        <test-method status="PASS" signature="testSetUp()[pri:0, instance:teammates.test.automated.EvaluationOpeningReminderTest@fc8157]" name="testSetUp" is-config="true" duration-ms="0" started-at="2014-08-07T04:58:33Z" finished-at="2014-08-07T04:58:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method status="PASS" signature="testTearDown()[pri:0, instance:teammates.test.cases.logic.EvaluationsLogicTest@7cb926]" name="testTearDown" is-config="true" duration-ms="219" started-at="2014-08-07T05:00:16Z" finished-at="2014-08-07T05:00:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTearDown -->
        <test-method status="PASS" signature="testTearDown()[pri:0, instance:teammates.test.automated.EvaluationOpeningReminderTest@fc8157]" name="testTearDown" is-config="true" duration-ms="0" started-at="2014-08-07T05:00:17Z" finished-at="2014-08-07T05:00:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTearDown -->
      </class> <!-- teammates.test.cases.logic.EvaluationsLogicTest -->
    </test> <!-- parallel-ui-tests(failed)(failed)(failed)(failed)(failed) -->
  </suite> <!-- Failed suite [Failed suite [Failed suite [Failed suite [Failed suite [all-tests-in-parallel]]]]] -->
</testng-results>
